README.md

```diff
@@ -132,16 +132,81 @@ SaaSKit comes with `primary` and `secondary` colors predefined within
 
 ## Deploying to Production
 
-### Authentication
+This section assumes that a
+[local development environment](#getting-started-locally) has been set up.
 
-TODO
+### Database (Supabase)
 
-### Customer Portal Branding (Stripe)
+In order to setup Supabase for production, you'll have to create two tables:
 
-[Set up your branding on Stripe](https://dashboard.stripe.com/settings/branding),
-as the user will be taken to Stripe's checkout page when they upgrade.
+- `todos`
+- `customers`
+
+#### Setup the `todos` table
+
+- Go to `Databases` > `Table Editor`
+- Click `New Table`
+- Enter the name as `todos` and check `Enable Row Level Security (RLS)`
+- Configure the following columns:
+
+| Name      | Type   | Default value        | Primary |
+| --------- | ------ | -------------------- | ------- |
+| `id`      | `uuid` | `uuid_generate_v4()` | `true`  |
+| `name`    | `text` | `NULL`               | `false` |
+| `user_id` | `uuid` | `auth.uid()`         | `false` |
+
+- Click the link symbol next to the `user_id` column name. Then, select schema
+  `auth`, table `users`, and column `id`. Now the `user_id` will link back to a
+  user object in Supabase Auth.
+
+#### Setup the `customers` table
+
+- Go to `Database` > `Tables`
+- Click `New Table`
+- Enter the name as `customers` and check `Enable Row Level Security (RLS)`
+- Configure the following columns:
+
+| Name                 | Type   | Default value | Primary |
+| -------------------- | ------ | ------------- | ------- |
+| `user_id`            | `uuid` | `auth.uid()`  | `true`  |
+| `stripe_customer_id` | `text` | `NULL`        | `false` |
+| `is_subscribed`      | `bool` | `false`       | `false` |
+
+- Click the link symbol next to the `user_id` column name. Then, select schema
+  `auth`, table `users`, and column `id`. Now the `user_id` will link back to a
+  user object in Supabase Auth.
+
+### Authentication (Supabase)
 
-### Webhooks for Subscriptions (Stripe)
+In your [Supabase dashboard](https://app.supabase.com/projects):
+
+1. Go to your project
+2. Go to `Authentication` > `Providers` > click `Email`
+3. Disable `Confirm email`
+4. Back on the left-hand bar, click on `Policies`
+5. Click `New Policy` on the `customers` table pane and then
+   `Create a policy from scratch`
+6. Enter the policy name as `Enable all operations for users based on user_id`
+7. For `Allowed operation`, select `All`
+8. For `Target Roles` select `authenticated`
+9. Enter the `USING expression` as `(auth.uid() = user_id)`
+10. Enter the `WITH CHECK expression` as `(auth.uid() = user_id)`
+11. Click `Review` then `Save policy`
+12. Repeat steps 5 to 11 for the `todos` table pane
```

Comment:
`supabase db push` takes care of this, too, so these lines aren't needed. Disabling confirm email part should stay, though.

Reply:
See [my previous comment](https://github.com/denoland/saaskit/pull/109#discussion_r1174601355)